apps/meteor/server/models/raw/Subscriptions.ts

```diff
@@ -594,6 +605,19 @@ export class SubscriptionsRaw extends BaseRaw<ISubscription> implements ISubscri
 		return this.updateMany(query, update);
 	}
 
+	updateNameAndFnameByVisitorIds(visitorIds: string[], name: string): Promise<UpdateResult | Document> {
+		const query = { 't': 'l' as const, 'v._id': { $in: visitorIds } };
```

Comment:
You can remove the `t` part from here, don't think mongo gets any extra improvements with it.